### DISCUSSION TITLE
ClaimBlocks per World

### DISCUSSION THREAD

**Discussion by TiberiuAnghel** [GriefPrevention/GriefPrevention#1284] [Q&A]
I've been looking recently for a option that allows you to set claims per world for example I have 2 survival worlds and I want to have different claim blocks on those 2 world and another question where I can set that players can't claim blocks but I am still able to claim as a admin

**Comment by Jikoo** [comment]
Re: separate claim block totals per-world: That's not a feature of GP right now. It's theoretically possible via an addon, but I don't know of any that do. If you really want your survival worlds separated that hard, you might consider running a Bungee (or other proxy) network with two separate survival servers behind it.

Re: creating admin claims without allowing players to claim: Enable claims but deny players the permission to make claims.
